docs/intro/reusable-apps.txt

```diff
@@ -76,7 +76,7 @@ After the previous tutorials, our project should look like this:
             static/
                 polls/
                     images/
-                        background.gif
+                        background.png
```

Comment:
I agree this needs fixing to match the tutorial06 section. Shal we split it in a separated PR?